snapcraft/tests/test_plugin_copy.py

```diff
@@ -332,6 +332,10 @@ def test_copy_symlinks_that_should_be_followed(self):
                 with open(destination, 'r') as f:
                     self.assertEqual(f.read(), symlink['expected_contents'])
 
+    def test_copy_enable_cross_compilation(self):
+        c = CopyPlugin('copy', self.mock_options, self.project_options)
+        c.enable_cross_compilation()
```

Comment:
this test does not verify anything. Maybe call `.build()` after or add a code comment about the expected behavior